exp/topology/desiredstate/desired_state_test.go

```diff
@@ -46,7 +46,7 @@ import (
 	"sigs.k8s.io/cluster-api/internal/hooks"
 	fakeruntimeclient "sigs.k8s.io/cluster-api/internal/runtime/client/fake"
 	"sigs.k8s.io/cluster-api/internal/topology/clustershim"
-	"sigs.k8s.io/cluster-api/internal/topology/names"
+	topologynames "sigs.k8s.io/cluster-api/internal/topology/names"
```

Comment:
Is it worth enforcing this import consistently by adding it to `.golangci.yml`?

https://github.com/kubernetes-sigs/cluster-api/blob/526ea91bb637c0f53a128496df005cae99a42a1e/.golangci.yml#L168-L171

Reply:
I think it is worth it because the CAPI project also uses "k8s.io/apiserver/pkg/storage/names" in a few places. Having distinct aliases for these imports improves clarity and avoids confusion between the two.

Reply:
Resolving as the alias was added to .golangci.yml. In general we usually only added frequently used aliases there. I guess this one is not used that much, but fine either way :)